knees4bees/rancid-tomatillos
# Fix/refactor app

## Is this a fix, feature, or refactor?
This is a refactor

## What is the change?
* A new main component has been built to handle main page view
* Moved error handling to the utilities.js file
* Fixed linting errors
* Adjusted state when clicking home button

## What does it fix?
This refactors the render method in the app component.  
Conditional rendering now happens in Main.js 
A method has been built to display errors or main page based on fetch errors being present.

## Where should the reviewer start?
Review the new Main.Js file then utilities.js to see new error handling function.

## How should this be tested?
Test in the browser. Uncomment out the last .then method in displayMovieDetails and click on movie posters to see the error message on the web  app.

## Relevant ScreenShots
